examples/compiled/test_subobject_nested.vg.json

```diff
@@ -84,17 +84,17 @@
             "transform": [
                 {
                     "type": "formula",
-                    "expr": "toNumber(datum[\"source\"][\"reco\"])",
+                    "expr": "toNumber(datum[\"source\"] && datum[\"source\"][\"reco\"])",
```

Comment:
if source is missing, what this does output?

Reply:
`null`

<img width="232" alt="screen shot 2018-04-29 at 10 59 01" src="https://user-images.githubusercontent.com/589034/39409461-5d7e52d2-4b9c-11e8-9499-38f49aff51a8.png">